gallery/plot_transforms.py

```diff
@@ -46,6 +46,29 @@ def _plot(img, title, **kwargs):
 resized_img = T.Resize(size=30)(orig_img)
 plot(resized_img, "Resized image")
 
+####################################
+# CenterCrop
+# ----------
+# The :class:`~torchvision.transforms.CenterCrop` transform
+# (see also :func:`~torchvision.transforms.functional.center_crop`)
+# Crops the given image at the center.
+center_cropped_img = T.CenterCrop(size=(100, 100))(orig_img)
+plot(center_cropped_img, "Center Cropped Image")
```

Comment:
Nit: perhaps "Center crop" would be simpler? (same for the other 5 ones). No strong opinion though!

Reply:
Let's keep consistency since all use `_img` word I will add it every place.

Reply:
I was mostly talking about the image title but I agree consistency is important!